modules/core/04-channel/keeper/upgrade.go

```diff
@@ -760,7 +760,7 @@ func (k Keeper) validateSelfUpgradeFields(ctx sdk.Context, proposedUpgrade types
 	currentFields := extractUpgradeFields(currentChannel)
 
 	if reflect.DeepEqual(proposedUpgrade, currentFields) {
-		return errorsmod.Wrapf(types.ErrChannelExists, "existing channel end is identical to proposed upgrade channel end: got %s", proposedUpgrade)
+		return errorsmod.Wrapf(types.ErrChannelExists, "existing channel end (%s) is identical to proposed upgrade channel end (%s)", currentFields, proposedUpgrade)
```

Comment:
this string makes it sound like we are displaying the channel end as opposed to the upgrade fields. 

Also due to the nature of the check, both values will always be the same, I don't think we need to display it twice.